### DISCUSSION TITLE
installation of kali in Mobile

### DISCUSSION THREAD

**Discussion by Harish-2003** [TermuxArch/TermuxArch#57] [General]
how to use my mobile as an linux computer

**Comment by ComeradeCharon** [comment]
Yes that's possible .....
You have to simply go on the kali linux official website and you can download the linux os file as per your mobile os capability.....
You can find all the details there in the website.....I wish you a very good luck 👍👍

**Comment by SDRausty** [comment]
>installation of kali in Mobile

Kale alternatives are ArchStrike and BlackArch,  and are documented  https://github.com/TermuxArch/docsTermuxArch here.

>how to use my mobile as an linux computer

The TermuxArch default install should fulfill this request.

**Comment by Harish-2003** [comment]
hey
